loopout.go

```diff
@@ -223,7 +232,8 @@ func resumeLoopOutSwap(reqContext context.Context, cfg *swapConfig,
 	log.Infof("Resuming loop out swap %v", hash)
 
 	swapKit := newSwapKit(
-		hash, swap.TypeOut, cfg, &pend.Contract.SwapContract,
+		pend.Contract.ProtocolVersion, hash, swap.TypeOut, cfg,
```

Comment:
This PR really has two changes: store protocol version and add htlc v2 (dependent on stored protocol version). You could order the commits in the same way.

Reply:
makes sense, ptal